src/main/java/frc/robot/subsystems/drive/Gearbox.java

```diff
@@ -1,11 +1,11 @@
-
 package frc.robot.subsystems.drive;
 
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANError;
 import com.revrobotics.CANSparkMax.IdleMode;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class Gearbox {
+public class Gearbox extends SubsystemBase{
```

Comment:
Since we don’t use the gearboxes directly in a command, does it need to extend SubsystemBase?